### DISCUSSION TITLE
Setting up accounts for regulatory compliance

### DISCUSSION THREAD

**Discussion by harshpatgi** [deepgram/community#358] [General help] [answered]
I am hoping to discuss setting up an account to ensure our compliance with regulatory requirements, specifically HIPAA, and to obtain a Business Associate Agreement.

**Comment by Deepgram-will** [comment, marked as answer]
For questions about account creation for regulatory compliance, please email sales@deepgram.com.

**Reply by harshpatgi** [reply]
Done! Thank you!

**Reply by cowanAI** [reply]
@Deepgram-will hello, same here, we dont require that but we tried paying using even different bank accounts and it seems the system is down, please let me know how can we submit a payment.

**Comment by lukeocodes** [comment]
@cowanAI sounds like you have a totally different issue. Are you saying there was an issue related to Stripe?

**Comment by cowanAI** [comment]
Correct, I was able to fix it by using my Polish bank card, for some reason
my American cards didn’t work, they have funds and it kept saying
insufficient funds.

**Reply by lukeocodes** [reply]
I saw an issue on our Discord community with payments this weekend. There may have been a service disruption for a limited time. I'll speak to the time and see if they have any information. In the meantime, can you share your project ID with me and I'll ask to check your account for any issues as well.

**Reply by cowanAI** [reply]
@lukeocodes For sure, 6cb24f72-41e9-4d18-b25d-f6852130a082, It seems more like a geographical issue, is it possible to pay directly, we truly need to recharge our credits to avoid any outages in our services, thanks so much!

**Comment by harshpatgi** [comment]
Hi Luke,

I didn't have a payment issue but rather I'm trying to obtain a business
associate agreement from Deepgram so our product can be HIPAA compliant.

Can you please assist me with this?

Thanks,
Harsh
